frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -148,6 +149,15 @@ private ConfigManager(Arguments args) {
             prop.setProperty("PYTHON_EXECUTABLE", pythonExecutable);
         }
 
+        String defaultResponseTimeout = System.getenv("DEFAULT_RESPONSE_TIMEOUT");
```

Comment:
It doesn't seems important enough to make it an environment variable.

Reply:
True, but I was trying to keep it consistent with our configurations. This way, customers should be able to configure anything in the config.properties through either CLI or environment variables. This is similar to other configurations that we have.

Reply:
We should not provide multiple way to do the same thing.
All the options exposed to CLI or Env should have a strong justification.
Currently we provide 4 options in CLI:
mms-config-file: config.properties cannot config itself, this is must have
python: this information is dynamic, java can't detect by itself
models/model-store: this is for backward compatible.

Reply:
After discussion, I agree we don't need this as CLI/Environment variable. I will remove it for now. We can take it up when we get a requirement for it.